client/src/components/participant-details/track-graduation.js

```diff
@@ -179,7 +193,17 @@ export const TrackGraduation = (props) => {
                 }}
                 onSubmit={async (values) => {
                   setShowArchiveModal(false);
-                  await handleArchive(props?.participant?.id, openToast, dispatchFunction, values);
+                  const hiredStatus = props?.participant?.latestStatuses?.find(
+                    (status) => status.status === participantStatus.HIRED
```

Comment:
We can assume that of all current statuses returned, there is only one hired for a specific site, at most.